### ISSUE TITLE
Indicate if a deployment was done with a different (non-console) client

### ISSUE THREAD

**Issue by anilmurty** [akash-network/console-old#90]
**Is your feature request related to a problem? Please describe.**
Users sometimes attempt to modify deployments not done using console. Example https://discord.com/channels/747885925232672829/1053359341144653957/1089949113916850227
Users need to be able to quickly know (and ideally filter) which deployments were done using console vs. other clients

**Describe the solution you'd like**
I can think or two ways to handle this:
1. Add some messaging (could be an icon or label next to the deployment name) -- this is the easiest but doesn't allow sorting or filtering)
2. Column to indicate if a deployment was not done using console - a "check" or something indicates that a different client was used - probably a little more work but will let the user sort and quickly see all deployments done with and without console
3. Filtering option (when we do support filters on this page).

FINAL DECISION/ DESIGN (ignore the "filter" and "view" UI):

![image](https://user-images.githubusercontent.com/19495789/232118595-83430e12-d648-4ad2-a1f3-3aa7ebd891dd.png)

** Implementation hint **
Check to see if we have the SDL. If we do then Console did the deployment. If not, some other client did.

**Describe alternatives you've considered**
Do nothing.

**Additional context**
There are multiple (vertically focused) clients for deploying on Akash: https://github.com/akash-network/community/tree/main/sig-clients

**Comment by aktdenis** [comment]
<img width="642" alt="Screenshot 2023-03-28 at 16 49 16" src="https://user-images.githubusercontent.com/116030971/228277458-430f3aae-ad1c-4446-8f5f-7457d8a3e870.png">
Additional label might work. Attaching a draft for "Certificates" where one indicates the label was created on a different computer - my suggestion is to create one solution to serve two purposes. (indicating a remote certificate, indicating a different deployment tool)

**Comment by anilmurty** [comment]
@aktdenis - thanks, that's an interesting idea but I fear that it may confuse users into thinking that the deployment is "remote" as in runs on a server not owned by them (which is not the intention)

**Comment by anilmurty** [comment]
@aktdenis - could you please make it a separate column for now, called "Updatable?"  and the value would be either a check mark icon or empty. And also we can add an "info" thing  next to "updatable" to say "If the initial deployment was done using a different client (not console) it cannot be updated by Console."

**Comment by aktdenis** [comment]
@anilmurty i'm thinking we should play around with wording. Maybe "static" or "non-updatable" to clearly communicate that the deployment cannot be changed. In contrast "updatable" may be misinterpreted or cause confusion. 

I'm an advocate for creating an additional label in "status" column - because that column already provides important information about the current state of the deployment. I believe this approach would help reduce visual clutter. 

Attaching 3 examples for further discussion:
- additional column "updatable + info/tooltip"
<img width="740" alt="Screenshot 2023-04-14 at 16 47 16" src="https://user-images.githubusercontent.com/116030971/232077644-f94e8eda-dde8-426f-95dd-8f6273894351.png">

- label within "status" column named: "non-updatable + lock icon" + info
<img width="587" alt="Screenshot 2023-04-14 at 16 47 08" src="https://user-images.githubusercontent.com/116030971/232077625-7ff57ae0-aca9-43ec-b5b6-0c13ffea07f5.png">

- label within "status" column named: "static + lock icon" + info/tool tip 
<img width="740" alt="Screenshot 2023-04-14 at 16 47 16" src="https://user-images.githubusercontent.com/116030971/232077607-d437a380-954d-4521-807a-5e3a2675ab3d.png">

@jtary feel free to chime in, so we can make the final decision.

**Comment by anilmurty** [comment]
Thanks @aktdenis. Did you mean to include a screenshot with the "updatable?" Column as well?

Re. The labels. I did think of that option too initially but decided not to go that route because the user wouldn't be able to sort and quickly group all non-console deployments together (we do not currently have filtering available). But ofcourse it wouldn't be a problem if we did have filtering supported.

**Comment by anilmurty** [comment]
Maybe one option is to go with the new column + "check" for now and then update to label when we can support filtering on this table

**Comment by aktdenis** [comment]
@anilmurty sorry! posted the wrong screenshot above. you had something like this in mind? <img width="712" alt="Screenshot 2023-04-14 at 16 46 59" src="https://user-images.githubusercontent.com/116030971/232106355-358415ee-ea74-42fe-aee8-f3fb90ca8c9e.png">

**Comment by anilmurty** [comment]
Yes - that's exactly what I had in mind @aktdenis . If it's ok with you I'd like to run with that for now and then move to your (nicer) design when we have filtering capabilities. Sound ok?

**Comment by aktdenis** [comment]
@anilmurty yes, sure!

**Comment by Amit0617** [comment]
~~Hi there, I wanted to know if that's how the implementation is supposed to work?~~
![Screenshot_20230417_133303](https://user-images.githubusercontent.com/71893015/232423358-9181131e-8e55-4257-836f-9a3ea16399e9.png)

~~Only we want a right check mark reading `info` state.~~

Edit: No, that doesn't seems to represent the updatable state.

**Comment by Amit0617** [comment]
Probably following things can help
```js
  const [sdl, setSdl] = useRecoilState(deploymentSdl);
```
and 
```js
          {
            label: 'Updatable',
            value: sdl ? 'Yes' : '',
          },
```
Would like to get your opinion.

**Comment by anilmurty** [comment]
@jtary - please review when you have a moment

**Comment by jtary** [comment]
@Amit0617 for the deployment to be updatable two things have to be true:

1. It has to be active (that's what the original check was looking for).
2. You have to have an SDL for it.

Technically, you don't _need_ the original SDL. You can update a deployment with any SDL as long as the group config matches (ie: same storage and compute requirements). You will have to update the deployment version as well, but that's generally a given unless you are updating with the exact same SDL.

For the sake of simplicity, we work off the assumption that we only can update if we have the original SDL. This so we can drop the user back into the SDL Configuration screen with the locked-in group requirements.

TL;DR:

```ts
canUpdateDeployment = deployment[deployment.state] === 'active' && sdl !== undefined;
```

Hope that helps.

**Comment by Amit0617** [comment]
I wasn't able to access that page in browser. I hope the above PR achieves the required.
